sdk/identity/azure-identity/tests/fake_credentials.py

```diff
@@ -0,0 +1,10 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+FAKE_CLIENT_ID = "fake-client-id"
```

Comment:
Was this triggering warnings (surprising because client IDs are not secret)?

Reply:
This wasn't, but I felt like a test client ID belonged more in this file than in `helpers.py` (since it's a component of constructing a credential) so I brought it over. I'm not strongly opposed at all to moving this back to the helpers file though!

Reply:
It's only pedantry making me write that it doesn't fit here. While I'm being pedantic though, nothing in this file is a credential, so this is more like `credscan_ignore.py`, and this constant still doesn't fit because credscan ignores it already. But I leave it up to you, I'm only driving by 💨

Reply:
Yeah, the naming is really awkward because you're right, nothing here is a "credential". The contents of the PowerShell strings just raise CredScan flags because of the contained token references. Your suggestion seems reasonable considering this file is more about satisfying CredScan than storing fake credentials centrally.